debian/control

```diff
@@ -15,7 +15,7 @@ Package: eos-watermark-extension
 Architecture: all
 Depends:
  gnome-shell (>= 3.35),
- gnome-shell (<< 3.37),
+ gnome-shell (>= 3.37.3),
```

Comment:
This should probably be <= 3.38 as done in the watermark extension?

Reply:
No because it cannot run on < 3.37.3